Source/IntegrationTests/TestFiles/LitTests/LitTest/unicodecharsFalse/comp/Arrays.dfy.rs.check

```diff
@@ -1 +1,3 @@
-// CHECK-L: Arrays.dfy(493,5): Error: Feature not supported for this compilation target: Multi-dimensional arrays
\ No newline at end of file
+
+Dafny program verifier finished with 0 verified, 0 errors
+(0,-1): Error: Microsoft.Dafny.UnsupportedInvalidOperationException: Creation of object of type MyClass requires a constructor
```

Comment:
I don't know what's going on here, so I think a better error message is in order.

Reply:
I think the error is thrown at [DafnyCodeGenerator.cs#L1525](https://github.com/dafny-lang/dafny/blob/3524aa17bb24ac206d2583425ddc594565881776/Source/DafnyCore/Backends/Dafny/DafnyCodeGenerator.cs#L1525), probably due to the empty class in [Arrays.dfy](https://github.com/dafny-lang/dafny/blob/ae6940829077b97f2487917e8b7c966fa6113109/Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/Arrays.dfy#L386); not sure what to do.

Reply:
Ok, your diagnosis seems reasonable. Can you see if it's not too much work to fix the token, so users get a source location to make sense of the message?